### PR TITLE
CASSANDRA-18969: source files missing from sources jars due to maven …

### PR DESCRIPTION
**Pull request by hhughes**
…include rule

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -221,3 +221,27 @@ see core/src/main/java/com/datastax/oss/driver/internal/core/util/CountingIterat
...
+which is available under the MIT License.
+see licenses/jnr-x86asm.txt
+
+This product bundles The Simple Logging Facade for Java (SLF4J) API,
+which is available under the MIT License.
+see licenses/slf4j-api.txt
+
+This product bundles Reactive Streams,
+which is available under the MIT License.
+see licenses/reactive-streams.txt
```
Comment (michaelsembwever) [inline LICENSE:247]: @hhughes , how come these are coming in ? (i thought we'd already included everything?)

Comment (hhughes) [reply LICENSE:247]: These are 3rd party binary jars included in the binary tarball we distribute.

Previously I was only thinking about the maven distribution mechanism where these dependencies would be pull in separately from the java driver jar.

**Comment by michaelsembwever** [comment]
Here's the commit (PR on top) to add the required checksums to the distribution* artefacts
https://github.com/hhughes/java-driver/pull/1

**Comment by hhughes** [comment]
@michaelsembwever done

**Comment by michaelsembwever** [comment]
I'll try to wrap up the review tomorrow, and if it looks ok merge and cut and stage a release.  That will help us get eyes on anything in the release we're missed…

**Comment by michaelsembwever** [comment]
Committed with some small addition changes as https://github.com/apache/cassandra-java-driver/commit/f11622308d031bf85047c4811e737aeb6ae236e9

**Pull request closed without merging**